bitnami/discourse/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```

Comment:
Check the Magento notes and add something similar here. We want to warn the users that they should upgrade their deployments if they didn't set the hostname. It makes sense to deploy it in two steps so they get the LoadBalancer IPs from the cloud provider.

Reply:
Ping on this one